plane/src/typed_unix_socket/mod.rs

```diff
@@ -15,10 +15,10 @@ pub struct WrappedMessage<T> {
 
 fn get_quick_backoff() -> ExponentialBackoff {
     ExponentialBackoff::new(
-        Duration::try_milliseconds(10).expect("duration is always valid"),
-        Duration::try_milliseconds(100).expect("duration is always valid"),
+        Duration::milliseconds(10),
```

Comment:
TIL that `Duration::milliseconds` was [un-deprecated](https://github.com/chronotope/chrono/releases/tag/v0.4.36), nice!